src/api/gameTitles/gameTitles.service.js

```diff
@@ -17,8 +26,57 @@ async function getByProductId(productId) {
   return gameTitles;
 }
 
+async function createTitle(newTitleData) {
+  const {
+    title, description, image, genre,
```

Comment:
What type of data is "genre"? What happens if the gameTitle has multiple genres?

Reply:
Good questions!
The route only accepts one genre, but a lot of titles have multiple genres, so we need to make the route capable of that.
Currently, genre is a string (comming from a form on the front-end). I think it should be an array of strings.
I'll work on it.